airbyte-integrations/connectors/source-postgres/src/main/resources/internal_models/internal_models.yaml

```diff
@@ -17,7 +17,7 @@ definitions:
     description: Enum to define the sync mode of state.
     type: string
     enum:
-      - cursorBased
+      - cursor_based
```

Comment:
Yes.
Discussed with Duy.
We user `one_two` in other two words keys/values. Camel case seems off.
This will affect only a [single customer](https://cloud.airbyte.com/workspaces/295bf192-9fa7-4b25-ba04-d1855541c6fe/connections/676f445a-4123-4d87-9429-7a22486bb105/job-history#3154175::0) which I will manually take care of before deploying.

Reply:
ack. Created https://github.com/airbytehq/airbyte/issues/28324 to track.